zipkin-web/src/main/scala/com/twitter/zipkin/web/Handlers.scala

```diff
@@ -55,7 +53,7 @@ class Handlers(jsonGenerator: ZipkinJson, mustacheGenerator: ZipkinMustache, que
 
     def apply(response: Response) {
       response.setContentType(typ)
-      response.content = ChannelBuffers.wrappedBuffer(content)
+      response.content = Buf.ByteArray(content)
```

Comment:
have to admit that this is more complex than the existing code.. I think I'll leave it.